tests/ProxyManagerTest/Functional/LazyLoadingGhostFunctionalTest.php

```diff
@@ -4,8 +4,8 @@
 
 namespace ProxyManagerTest\Functional;
 
-use PHPUnit_Framework_MockObject_MockObject as Mock;
 use PHPUnit\Framework\TestCase;
+use PHPUnit_Framework_MockObject_MockObject as Mock;
```

Comment:
As far as I know, we can use the namespaced version here.

Reply:
Checked: will require a second patch targeting PHPUnit upgrade